ig/input/fsh/nbs-obx-segment.fsh

```diff
@@ -2,16 +2,32 @@ Logical: OBXSegment
 Id: segment-obx-logical-model
 Title: "OBX Segment"
 Description: "The OBX Segment"
-* setOBXId 1..1 SU string "OBX-1 (1) Birthweight an identifier"
-* valueType 1..1 SU string "OBX-2 A Value Type"
-* observationIdentifier 1..1 SU string "OBX-3 Observation Identifier"
-* observationSubID 1..1 SU string "OBX-4 (LOINC 54089-8) Observation Sub-ID"
-* observationValue 1..1 SU string "OBX-5 (LOINC or 99MDH) Observation Value"
-* units 1..1 SU string "OBX-6 Unit For Observation Value"
-* observationResultStatus 1..1 SU string "OBX-11 Observation Result Status"
-* dateTimeOfTheObservation 1..1 SU string "OBX-14 Date/time Of Observation"
-* observationType 1..1 SU string "OBX-29 Observation Type"
-* observationSubType 1..1 SU string "OBX-30 Observation Sub-type"
+* setOBXId 1..1 string "OBX-1 (1) Birthweight an identifier"
+* valueType 0..1 id "OBX-2 A Value Type"
+* observationIdentifier 1..1 code "OBX-3 Observation Identifier"
+* identifier 1..1 string "OBX-3.1 Identifier"
+* observationtText 1..1 string "OBX-3.2 Text"
+* observationNameOfCodingSystem 1..1 id "OBX-3.3 Name of Coding System"
+* observationSubID 1..1 string "OBX-4 (LOINC 54089-8) Observation Sub-ID"
+* observationValue 1..1 string "OBX-5 (LOINC or 99MDH) Observation Value"
+* observationValueUnits 1..1 code "OBX-6 Units (Observation Value)"
+* obxValueUnitsIdentifier 1..1 code "OBX-6.1 (Unit) Identifier"
+* obxValueUnitsText 1..1 code "OBX-6.2 (Unit) Text"
+* obxValueUnitsNameOfCodingSystem 1..1 code "OBX-6.3 (Unit) Name of Coding System"
+* observationReferenceRange 1..1 string "OBX-7 Observation Reference Range"
```

Comment:
Even though we may have the mapping for these additional OBX fields (OBX-7, OBX-8 and OBX-15) because we got them from ORU, I'm not sure if we should document them until we actually have them in an OML message and have verified that the conversion is working for those